content/pages/conference/index.rst

```diff
@@ -0,0 +1,18 @@
+===========
+Conference
+===========
+
+:date: 2023-09-28 09:00
+:url: conference
+:save_as: conference/index.html
+
+sicwork 2023
+================================================
+    
+`sciwrok 2023 <https://conf.sciwork.dev>`__ is to provide a physical venue to showcase how programming languages are 
+used for scientific and engineering applications and their best practices. In addition 
+to scientific computing and high-performance computing, we also emphasize data management, 
+process, analytics, and visualization.
+
+- Date: December 9-10, 2023
+- Location: Location: Microelectronics and Information Research Center, NYCU, Hsinchu, Taiwan
```

Comment:
`Location:` is duplicated.  Please fix it.

Reply:
Thanks. Fix it with #266 and publish.